tests/drivers/display/display_read_write/testcase.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 (c) TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  tags:
+    - drivers display
+  filter: dt_chosen_enabled("zephyr,display")
+  timeout: 1000
+  build_only: true # The CI environment has no display device
+tests:
+  drivers.display.read_write.sdl.argb8888:
+    filter: dt_compat_enabled("zephyr,sdl-dc")
+    platform_exclude: native_posix native_posix_64
```

Comment:
Any reason to exclude these ones?
(note that if the aim was to limit how many tests are run in CI, and to focus on native_sim by default, you can use `integration_platforms: native_sim`)

Reply:
I forgot to remove this line.
I also enabled the `build_only` flag at this time because this driver can't be used in the CI environment.
So, I will reconsider the test targets when the driver can run in CI environment.